dash_labs/plugins/pages.py

```diff
@@ -259,12 +277,20 @@ def router():
         def update(pathname, search):
             path_id = app.strip_relative_path(pathname)
             query_parameters = _parse_query_string(search)
+
             layout = None
             for module in dash.page_registry:
                 page = dash.page_registry[module]
+
                 if path_id == app.strip_relative_path(page["path"]):
                     layout = page["layout"]
 
+                if page["path_template"]:
+                    template_id = app.strip_relative_path(page["path_template"])
+                    path_variables = _parse_path_variables(path_id, template_id)
+                    if path_variables:
+                        layout = page["layout"]
```

Comment:
🐎 do we want to `break` here and [above](https://github.com/plotly/dash-labs/pull/61/files#diff-3242435debccc1e6c9e9164d3826cd066d02302d12cda2cb20956e71d27da55eR286) since we found a matching page?

Reply:
For sure - that would be more efficient

Reply:
actually, I could add a `break` here but not [above](https://github.com/plotly/dash-labs/pull/61/files#diff-3242435debccc1e6c9e9164d3826cd066d02302d12cda2cb20956e71d27da55eR286) because above can be true for paths with templates